### DISCUSSION TITLE
seed data from json files where it makes sense

### DISCUSSION THREAD

**Discussion by dalelotts** [epicweb-dev/epic-stack#869] [Ideas]
Consider adding the ability to seed data from json files, I find it much easier than writing code to seed data.

**Comment by kentcdodds** [comment]
You could definitely change the seed script to import a json file and use that for seeding. I personally really like the ability to dynamically change how much data is seeded and generate the seed data easily using faker. This is something I do not think I will do, but as usual I welcome an [example](https://github.com/epicweb-dev/epic-stack/blob/main/docs/examples.md) if you're interested in making one!

**Comment by dalelotts** [comment]
I've already done this for my own project for seeding prod, where faker isn't what I want, but a migration works well for this also.

**Reply by kentcdodds** [reply]
If you need data in prod, doing it in the migration is almost always the right way to go.
